app/controllers/katello/api/v2/host_subscriptions_controller.rb

```diff
@@ -35,6 +37,42 @@ def events
       respond_for_index :collection => collection
     end
 
+    api :POST, "/hosts/subscriptions/register", N_("Simulate a host subscription-manager registration")
```

Comment:
I'm still not sold. We are replicating an API, it says so right in the description text using a synonym. Are we planning to allow host creation through every facet? How or why can the RHSM API not do what this replication of the RHSM API can do?